app/client/src/constants/ApiConstants.tsx

```diff
@@ -15,6 +15,13 @@ export enum API_STATUS_CODES {
   REQUEST_NOT_AUTHORISED = 401,
   RESOURCE_NOT_FOUND = 404,
   SERVER_ERROR = 502,
+  SERVER_UNAVAILABLE = 503,
+}
+
+export enum ERROR_CODES {
+  PAGE_NOT_FOUND = 404,
+  SERVER_ERROR = 502,
+  REQUEST_TIMEOUT = "ECONNABORTED",
```

Comment:
Keeps it separate from API level codes
```suggestion
export enum ERROR_CODES {
  PAGE_NOT_FOUND = "PAGE_NOT_FOUND",
  SERVER_ERROR = "SERVER_ERROR",
  REQUEST_TIMEOUT = "REQUEST_TIMEOUT",
```